docs/table-of-contents.md

```diff
@@ -0,0 +1,6 @@
+
+# Material2 Table of Contents
```

Comment:
Shouldn't this be "Material" or just "Table of Contents"?

Reply:
yeah, something like that, either way, a docs folder or examples folder would help new users

Reply:
So your PR is not ready yet?

Reply:
this PR is just a concept, if you like the idea, I can fill it out